radionice/pocetni/lekcije/004_osobne_zamjenice_i_oslovljavanje/004_osobne_zamjenice_i_oslovljavanje.tex

```diff
@@ -53,63 +56,66 @@
 	\begin{reibun}
 		\rei{\underline{あなた}、すいか　が　すき？\footnotemark[1]}{Ti voliš lubenice?}
 	\end{reibun}
-
-	\footnotetext[1]{Ovo zvuči kao da žena pita muža - \textit{Dragi, voliš li lubenice?}}
 	
+	\begin{table}
 	\begin{tabular}{|l|l|l|}
 		\hline
 		かれ&on, dečko&neutralni muški\\\hline
 		かのじょ&ona, cura&neutralni ženski\\\hline
 	\end{tabular}
-	
-	\vspace{10pt}
-	
+	\end{table}
+
+	\footnotetext[1]{Ovo zvuči kao da žena pita muža - \textit{Dragi, voliš li lubenice?}}
+	\newpage
+
 	かれ　i　かのじょ　koristimo kada pričamo o trećoj osobi čije ime neznamo ili nam je ta osoba glavna tema priče pa ime ne moramo ponavljati. Nikad ne koristimo za osobe koje ne poznamo jer izražavaju bliskost. Umjesto toga koristimo あのかた、あのひと...Osim toga, かれ　i　かのじょ može značiti "cura" i "dečko" u intimnom smislu.
 	
 	\begin{reibun}
 		\rei{\underline{かれ}　の　にっき}{njegov dnevnik}
 		\rei{\underline{わたし}　の　\underline{かのじょ}　の　にっき}{dnevnik moje cure}
 	\end{reibun}
-	
+
+	Izražavanje drugog lica jednine(ti) na pogrdne ili neprikladne načine: 
+	\begin{table}[!h]
 	\begin{tabular}{|l|l|l|}
 		\hline
-		そなた&ti&pjesnički „ti“, arhaizam\\\hline
-		きみ\footnotemark[2]&ti&koristi se za M i Ž pogotovo kad se stariji obraća puno mlađem od sebe\\\hline
-		あんた&ti&arogantniji, češće koriste Ž nego M\\\hline
-		おまえ&ti&pogrdno, ponekad se koristi među muškim prijateljima, M\\\hline
-		てめぇ&ti&pogrdno, M i Ž\\\hline
-		きさま&ti&pogrdno, nekoć bio izraz keiga, M i Ž\\\hline
-	\end{tabular}	
-	
-	\vspace{10pt}
+		そなた			&M i Ž	&pjesnički „ti“, arhaizam\\\hline
+		きみ\footnotemark[2]	&M i Ž	&primarno se koristi kad se stariji obraća puno mlađem od sebe\\\hline
+		あんた			&Ž	&arogantniji, češće koriste Ž nego M\\\hline
+		おまえ			&M	&pogrdno, ponekad se koristi među muškim prijateljima\\\hline
+		てめぇ			&M i Ž 	&pogrdno\\\hline
+		きさま			&M i Ž 	&pogrdno, nekoć bio izraz keiga\\\hline
+	\end{tabular}
+	\end{table}
 	
 	\footnotetext[2]{Osoba koja govori きみ je iznad osobe za koju se to izgovara, ponekad きみ koriste muške osobe kada se obraćaju svojoj jako bliskoj prijateljici}
 	
 		Ako razgovaramo sa osobom koja nam je inferiorna ili na istoj razini tada možemo koristiti inferiorne osobne zamjenice, uz napomenu da one mogu zvučati poprilično nepristojno i uvrijediti osobu.
```

Comment:
Ovaj tekst je nezgodno sročen - što znači da nam je netko inferioran? Još teže, što to znači u japanskoj kulturi? Trebali bismo se malo bolje izraziti.

Reply:
Mozda netko koga smatramo inferiornom. Sad je promijenjeno u to